redbeams/src/main/java/com/sequenceiq/redbeams/service/dbserverconfig/DatabaseServerConfigService.java

```diff
@@ -76,7 +78,7 @@
     @Inject
     private UserGeneratorService userGeneratorService;
 
-    public Set<DatabaseServerConfig> findAll(Long workspaceId, String environmentId, Boolean attachGlobal) {
+    public Set<DatabaseServerConfig> findAll(Long workspaceId, String environmentId) {
```

Comment:
We don't use workspaceId in microservices as well

Reply:
Yes. It remained there because of historic reasons ... I do not want toe delete it as part of this jira but we could get rid of it. @bhavanki, what do you suggest?

Reply:
At this point, I'm fine getting rid of it. It's becoming more of a wart.